misc/scripts/install/docker-volume-vsphere.conf

```diff
@@ -7,4 +7,4 @@ respawn
 
 kill timeout 20
 
-exec /usr/local/bin/$UPSTART_JOB
+exec /usr/local/bin/$UPSTART_JOB -port 1019
```

Comment:
This helps in debugging cause the PS lists the port.. considering the difficulty we have been having with tracking sockets and that the code is shipped as binary.